plugins/modules/ec2_transit_gateway.py

```diff
@@ -181,6 +188,12 @@
           returned: always
           type: str
           sample: enable
+        multicast_support:
+          description: Indicates whether Multicast support is enabled.
+          returned: always
+          type: str
```

Comment:
Isn't it supposed to be a `bool` ?

Reply:
> Isn't it supposed to be a `bool` ?

To configure is a bool, but return value is a string, sample "enable". https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_TransitGatewayRequestOptions.html